packages/component/src/lib/embedding_view/EmbeddingViewImpl.svelte

```diff
@@ -2,9 +2,9 @@
 <script lang="ts" module>
   interface Props<Selection> {
     data: {
-      x: Float32Array;
-      y: Float32Array;
-      category: Uint8Array | null;
+      x: Float32Array<ArrayBuffer>;
```

Comment:
So we don't want to support shared array buffers?

Reply:
WebGPU doesn't seem to allow shared array buffers.